README.md

```diff
@@ -8,5 +8,44 @@ The tutorials found here are _shared tutorials_ that complement more specific tu
 
 Please check out the [learn](https://scverse.org/learn) page on scverse.org for an overview of all tutorials!
 
+# tutorial registry
+
+The purpose of this document is to formalize the criteria to be eligible for external tutorial inclusion into the scverse tutorials listing (https://scverse.org/scverse-tutorials/).
+
+## Criteria to be eligible for tutorial inclusion
+
+We have the following requirement for a package to be eligible to be considered for tutorial submission/mirroring:
+
+1. The package must be a scverse listed ecosystem package and therefore meet all corresponding requirements.
+2. Any used (advanced) methods must be published. Preprints are acceptable on a case by case basis.
```

Comment:
It's not quite clear what "on a case by case basis" here means. Is it us judging this? Should we ask people with preprints to contact us beforehand ?